README.md

```diff
@@ -32,6 +22,16 @@ This operator was created using the
 [operator-sdk](https://github.com/operator-framework/operator-sdk/). It's not
 strictly required but does provide some handy tooling.
 
+### Knative Serving
+
+The following will install [Knative Serving](https://github.com/knative/serving)
+and configure it appropriately for your cluster in the `knative-serving`
+namespace:
+
+```
+kubectl apply -f deploy/crds/serving_v1alpha1_knativeserving_crd.yaml
+```
```

Comment:
This won't actually install the operator. Do you need something like `ko apply -f deploy/` ?

Reply:
Right, I move that part downstairs. I changed this section into "Knative serving".